README.md

```diff
@@ -31,6 +31,148 @@ What modules you need to use SJORM
+```java
+SJORM.register("127.0.0.1", 3306, "database", "username", "password");
+SJORM.getInstance().registerModel(new ModelMetadata(Game.class));
+```
+
+The SJORM service is now aware that the `Game` class is a model that exists in your database.
+
+The static `SJORM.register()` method returns the new instance of the SJORM service. Meaning, it is possible to save it as a variable and access it from there as well instead of using the static `SJORM.getInstance()` method. Either way, both options return the same instance of the SJORM service.
+
+Setting up the SJORM service is now all done!
+
+### Retrieving models
+To query data from your database and get them as a collection of models, you can use the available methods in the SJORM service. One of the straight-forward methods is the `getAll()` method.
+```java
+Query<Game> games = sjorm.getAll(Game.class);
+```
+
+A `Query<T>` is a class in which you can query on a collection, and it contains instances of your pre-defined models. It contains methods that you can use to manipulate the entries in the collection. It is an equivalent and simplified version of LINQ from C# (<https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/>) and somewhat comparable to Stream that was released in Java 8. It makes use of method-chaining, meaning you can chain the methods together and call them one after another, while still returning a `Query<T>`.
+
+See some examples below on what you can do with a `Query<T>` instance:
+```java
+Query<Game> games = sjorm.getAll(Game.class);
+
+// Get the amount of games
+int gameAmount = games.count();
+
+// Get the first Game instance from the collection
+Game game = games.first();
+
+// Get a list of all names of the games
+List<String> gameNames = games.select("name");
+
+// Get a list of games ordered by id
+Query<Game> gamesOrderedById = games.orderBy("id");
+
+// Get all games where the name starts with an 'A'
+Query<Game> gamesStartingWithA = games.where(g -> g.getProperty("name", String.class).matches("[aA].*"));
+
+// Loop over all Game instances
+for(Game game : games.toList()) {
+    // Do something with 'game'
+}
+```
+
+A custom `Query<T>` can be instantiated from an instance of Java's `List<T>` class, by using the constructor that requires a `List<T>`, as shown below:
+```java
+List<String> names = new ArrayList<>(Arrays.asList("John", "Kim", "Danny"))
+        
+Query<String> queryNames = new Query(names);
+```
+
+To access the fields of a model, you will have to use its properties. The properties are stored in a `LinkedHashMap` that can be accessed by using either `Model#getProperty()` or `Model#setProperty()`. Where the one acts as a setter and the other as a getter. There are two variations of the `Model#getProperty()` method. One where you can get the value of the property as an `Object` type, and the other where you can specify an explicit type cast to get the property as its correct data type. See the example below:
```

Comment:
Change the wording here: `Where the one acts as a setter and the other as a getter` to: `Where the latter acts as a setter and the former as a getter`. "The one" doesn't make much sense here and does not make it clear which one you are referencing.